denizen_scripts/survival/repo-link/spawn_scripts/player_joins_server.dsc

```diff
@@ -1,7 +1,7 @@
 player_joins_server:
   type: world
   events:
-    on player logs in for the first:
+    on player logs in for the first time:
       - flag player first_joined:true
```

Comment:
This is an unreliable way to verify if a player joins the server for the first time. `player logs in` is during the authentication process, not to be confused with the `player join` event which fires when they actually physically make it to the server